Makefile

```diff
@@ -25,7 +25,7 @@ fmtcheck:
 	@sh -c "'$(CURDIR)/script/gofmtcheck.sh'"
 
 lint: fmtcheck
-	$(DOCKER) run --env=GOFLAGS=-mod=vendor --rm -v $(CURDIR):/app -w /app golangci/golangci-lint:v1.43.0 golangci-lint -v run -E tparallel,whitespace
+	$(DOCKER) run --env=GOFLAGS=-mod=vendor --rm -v $(CURDIR):/app -w /app golangci/golangci-lint:v1.43.0 golangci-lint -v run -E revive,tparallel,whitespace
```

Comment:
This will be fixed in a more general way with #466

Reply:
Done